influxql/parser.go

```diff
@@ -1476,6 +1476,11 @@ func (p *Parser) parseField() (*Field, error) {
 	if err != nil {
 		return nil, err
 	}
```

Comment:
Another idea: the `validField` visitor could store the operator and the error message could read something like `invalid operator < found in field expression at <position>`